### PR TITLE
chore(logging-apps): update helm-charts

### PR DESCRIPTION
**Pull request by maxthier**
# Description

This PR updates the helm-charts in the logging-apps chart.

# Issues

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I updated the version in Chart.yaml
* [x] I updated the changelog with an `artifacthub.io/changes` annotation in `Chart.yaml`, check the [example](docs/development.md#Changelog) in the documentation.
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [ ] CI is currently green and this is ready for review
* [ ] I am ready to test changes after they are applied and released

### CODE REVIEW

**Comment by maxthier** [comment]
Where can we test changes like fluentd? Also somehow we havent updated fluentd since 2023-03-18.

**Comment by tongpu** [comment]
> Where can we test changes like fluentd? Also somehow we havent updated fluentd since 2023-03-18.

I'm honestly think we're not using fluentd at any customers anymore.

File: charts/logging-apps/Chart.yaml [hunk 1]
```diff
@@ -18,7 +18,28 @@ annotations:
...
+      links:
+        - name: "promtail Release 6.16.0"
+          url: https://github.com/grafana/helm-charts/releases/tag/promtail-6.16.0
+    - kind: changed
+      description: "Update fluentd chart from 5.5.* to 6.5.*"
+      links:
+        - name: "fluentd changelog"
+          url: https://github.com/bitnami/charts/blob/main/bitnami/fluentd/CHANGELOG.md
+    - kind: changed
+      description: "Update fluentBit chart from 0.46.7 to 0.46.111"
```
Comment (hairmare) [inline charts/logging-apps/Chart.yaml:42]: 111 or 11?

**Comment by hairmare** [comment]
> > Where can we test changes like fluentd? Also somehow we havent updated fluentd since 2023-03-18.
> 
> I'm honestly think we're not using fluentd at any customers anymore.

yeah, it should be replaced with fluentbit at this point... we should follow up with deprecating this in the chart after this PR

File: charts/logging-apps/Chart.yaml (hunk 1 above)
Comment (maxthier) [inline charts/logging-apps/Chart.yaml:42]:
```suggestion
      description: "Update fluentBit chart from 0.46.7 to 0.46.11"
```

**Review by hairmare — APPROVED** [review]

**Pull request merged**